packages/http-client-python/eng/scripts/ci/run_pylint.py

```diff
@@ -47,4 +48,8 @@ def _single_dir_pylint(mod):
 
 
 if __name__ == "__main__":
+    if os.name == "nt":
+        # Before https://github.com/microsoft/typespec/issues/4759 fixed, skip running Pylint for now on Windows
+        logging.info("Skip running Pylint on Windows for now")
```

Comment:
We are in dilemma If we add pylint disable to fix for Windows, it will cause useless-suppressions info in Linux; if we do nothing, pylint fails in Windows.

Before https://github.com/microsoft/typespec/issues/4759 fixed, we have to skip pylint check in Windows. Is that OK? @iscai-msft / @tadelesh

Reply:
talked offline, we only run pylint on 3.8, so ideally we try to make sure pylint is run ith version 3.8, but I think this is fine too